specutils/analysis/flux.py

```diff
@@ -178,7 +177,8 @@ def is_continuum_below_threshold(spectrum, threshold=0.01):
     # we are going to calculate based on the S/N if the uncertainty
     # exists.
     if uncertainty and uncertainty.uncertainty_type != 'std':
-        return np.median(flux / uncertainty.quantity) < threshold
+        # return np.median(flux / uncertainty.quantity) < threshold
```

Comment:
```suggestion
```

Reply:
Should we use uncertainty.quantity, or just uncertainty here? The tests have no combination that checks this line of code.